### PR TITLE
[New] Animate 3D graphic

### PR DESCRIPTION
**Pull request by CalebRas**
## Description

This PR implements `Animate 3D graphic` in `Scenes` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-Animate3DGraphic/Shared/Samples/Animate%203D%20graphic)

## Linked Issue(s)

- `swift/issues/4586`

## How To Test

Tap the buttons on the bottom toolbar to adjust the settings for the animation:

- Mission: change the flight path, speed, and view progress
- Play/Pause: toggle the animation
- Camera: change the camera distance, heading, pitch, and other camera properties.

## Screenshots

animate-3d-graphic

### CODE REVIEW

File: Shared/Supporting Files/Extensions/View+Sheet.swift [hunk 1]
```diff
@@ -194,10 +194,11 @@ private extension SheetModifier {
...
             Sheet(
-                isPresented: Binding(
-                    get: { isPresented && !isPopoverVisible },
-                    set: { isPresented = $0 }
-                ),
+//                isPresented: Binding(
+//                    get: { isPresented && !isPopoverVisible },
+//                    set: { isPresented = $0 }
+//                ),
+                isPresented: $isPresented,
```
Comment (dfeinzimer) [inline Shared/Supporting Files/Extensions/View+Sheet.swift:201]: Was this change accidental?

File: Shared/Samples/Animate 3D graphic/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+![Image of animate 3D graphic](animate-3d-graphic.png)
+
+## Use case
+
+Visualize movement through a 3D landscape.
+
+## How to use the sample
+
+Animation Controls (Top Left Corner):
```
Comment (dfeinzimer) [inline Shared/Samples/Animate 3D graphic/README.md:13]: Should this indicate the bottom left corner instead?

**Review by dfeinzimer — COMMENTED** [review]
Overall this looks really. Good, just a few questions.

File: Shared/Supporting Files/Extensions/View+Sheet.swift (hunk 1 above)
Comment (CalebRas) [reply Shared/Supporting Files/Extensions/View+Sheet.swift]: Yes, thank you for catching this.

File: Shared/Samples/Animate 3D graphic/README.md (hunk 2 above)
Comment (CalebRas) [reply Shared/Samples/Animate 3D graphic/README.md]: I had forgotten to remove that.

File: Shared/Samples/Animate 3D graphic/Animate3DGraphicView.Model.swift [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+                // Reset the animation when it has reached the end.
+                reset()
+            } else {
+                // Move the index to point to the next frame.
+                currentFrameIndex += 1
+            }
+        }
+        
+        /// Loads the frames of a mission from a CSV file.
+        /// - Parameter filename: The name the file containing the CSV data.
```
Comment (pgruenler) [inline Shared/Samples/Animate 3D graphic/Animate3DGraphicView.Model.swift:282]:
```suggestion
        /// - Parameter filename: The name of the file containing the CSV data.
```

**Review by pgruenler — APPROVED** [review]
LGTM I just left a couple nit-picky comment suggestions.

**Review by pgruenler — APPROVED** [review]

**Review by dfeinzimer — APPROVED** [review]

**Pull request merged**